packages/client/tests/functional/handle-int-overflow/tests.ts

```diff
@@ -1,22 +1,18 @@
-import { getQueryEngineProtocol } from '@prisma/internals'
-
 import testMatrix from './_matrix'
 // @ts-ignore
 import type { PrismaClient } from './node_modules/@prisma/client'
 
 declare let prisma: PrismaClient
 
 testMatrix.setupTestSuite(() => {
-  testIf(getQueryEngineProtocol() !== 'json')('integer overflow', async () => {
+  test('integer overflow', async () => {
     await expect(
       prisma.entry.create({
         data: {
           int: 1e20,
         },
       }),
-    ).rejects.toThrow(
-      /A number used in the query does not fit into a 64 bit signed integer. Consider using `BigInt` as field type if you're trying to store large integers./,
-    )
+    ).rejects.toThrow(/Unable to fit value 100000000000000000000 into a 64-bit signed integer for field `int`/)
```

Comment:
would be nice to preserve the "consider using `BigInt`" part of the message

Reply:
https://github.com/prisma/team-orm/issues/216